motor/discovery/docker_engine/container.go

```diff
@@ -111,7 +111,11 @@ func (e *dockerEngineDiscovery) ImageInfo(name string) (ImageInfo, error) {
 	labels["docker.io/tags"] = strings.Join(res.RepoTags, ",")
 	labels["docker.io/digests"] = strings.Join(res.RepoDigests, ",")
 
-	ii.Name = containerid.ShortContainerImageID(res.ID)
+	if len(res.RepoTags) > 0 {
+		ii.Name = res.RepoTags[0]
+	}
+
+	ii.Name = ii.Name + "@" + containerid.ShortContainerImageID(res.ID)
```

Comment:
for images with no tag, wouldn't this lead to @ being prepended, i.e. `@edcf96f9d9d9`?

Reply:
Yes yes yes, great call. Let me fix that quick